CHANGELOG.md

```diff
@@ -2,6 +2,9 @@ Changelog for the gruf gem. This includes internal history before the gem was ma
 
 ### Pending release
 
+- Drop Ruby 2.2 support
```

Comment:
You need to update the version here we as well to 2.6.0

Reply:
Nope, it's still a pending release, no version # is set yet. You only do that on tags - hence the "Pending Release" header. That allows us flexibility on semver.

Reply:
Then why are we bumping the version in the version file https://github.com/bigcommerce/gruf/pull/71/files#diff-d78e8222a51844b601691817db97dd6dR17

Reply:
That's a good question - so the reason is we'll set on 2.6.0 because dropping Ruby 2.0 means we need to bump to _at least_ 2.6.0; however, that doesn't necessarily mean we'll end there (master, for example, might jump to 3.0).